src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultGeneratedDocumentPublisher.cs

```diff
@@ -106,9 +105,7 @@ public override void PublishCSharp(string filePath, SourceText sourceText, int h
                 HostDocumentVersion = hostDocumentVersion,
             };
 
-            var result = _server.SendRequest(RazorLanguageServerCustomMessageTargets.RazorUpdateCSharpBufferEndpoint, request);
-            // This is the call that actually makes the request, any SendRequest without a .Returning* after it will do nothing.
-            _ = result.ReturningVoid(CancellationToken.None);
+            _ = _server.SendNotificationAsync(RazorLanguageServerCustomMessageTargets.RazorUpdateCSharpBufferEndpoint, request, CancellationToken.None);
```

Comment:
FWIW, `Microsoft.VisualStudio.Threading` has a `FireAndForget` extension method. Is that available in this project? If so, you coule do this:

```suggestion
         _server.SendNotificationAsync(RazorLanguageServerCustomMessageTargets.RazorUpdateCSharpBufferEndpoint, request, CancellationToken.None).FireAndForget();
```

I assume that this is truly a "fire and forget" situation, so `CancellationToken.None` is OK?

Reply:
Does not appear to be available, but I like how that makes the intent more explicit.